src/poll-until-promise.ts

```diff
@@ -55,9 +60,11 @@ export class PollUntil {
     backoffFactor = 1,
     backoffMaxInterval,
     message = '',
+    maxAttempts,
   }:IWaitForOptions = {}) {
     this._interval = interval;
     this._timeout = timeout;
+    this._attemptCount = 1;
```

Comment:
it's a bit ambiguous regarding the "current" attempt.
maybe we should specify `executedAttempts`?
so it will be explicit as if the counter is at the "end" of the attempt.

Reply:
Yep, agree. Done